plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/TrinoGlueCatalog.java

```diff
@@ -344,6 +344,11 @@ public Table loadTable(ConnectorSession session, SchemaTableName table)
 
     @Override
     public void dropTable(ConnectorSession session, SchemaTableName schemaTableName)
+    {
+        dropTable(session, schemaTableName, true);
```

Comment:
alternatively, the code can be

```
unregisterTable(...);
dropTableData(table.io(), table.operations().current());
deleteTableDirectory(fileSystemFactory.create(session), schemaTableName, table.location());
```

without defining `dropTable(ConnectorSession session, SchemaTableName schemaTableName, boolean deleteData)` helper method

Reply:
(same in others)